src/handlers/swap_handlers.ts

```diff
@@ -45,12 +46,39 @@ import {
 
 import { quoteReportUtils } from './../utils/quote_report_utils';
 
+const BEARER_RE = /^Bearer\s(.{36})$/;
+const REGISTRY_SET: Set<string> = new Set(RFQT_REGISTRY_PASSWORDS);
+const REGISTRY_ENDPOINT_FETCHED = new Counter({
+    name: 'swap_handler_registry_endpoint_fetched',
+    help: 'Requests to the swap handler',
+    labelNames: ['identifier'],
+});
+
 export class SwapHandlers {
     private readonly _swapService: SwapService;
     public static rootAsync(_req: express.Request, res: express.Response): void {
         const message = `This is the root of the Swap API. Visit ${SWAP_DOCS_URL} for details about this API.`;
         res.status(HttpStatus.OK).send({ message });
     }
+
+    public static async getRfqRegistryAsync(req: express.Request, res: express.Response): Promise<void> {
+        const auth = req.header('Authorization');
+        if (auth === undefined) {
+            return res.status(HttpStatus.UNAUTHORIZED).end();
+        }
+        REGISTRY_ENDPOINT_FETCHED.labels(auth).inc();
```

Comment:
Maybe place the counter ahead of auth === undefined if we want to see all requests?

Reply:
good call!